src/utils/daos.c

```diff
@@ -1071,8 +1071,57 @@ print_oclass_names_list(FILE *stream)
 	free(str);
 }
 
+#define FIRST_LEVEL_HELP \
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon